src/lib/components/Modal/Modal.tsx

```diff
@@ -125,24 +95,18 @@ const ModalComponent: FC<ModalProps> = ({
   return createPortal(
     <ModalContext.Provider value={{ popup, onClose }}>
       <div
-        aria-hidden={!show}
```

Comment:
Am I understanding correctly that you deleted this line because the element is no longer rendered at all in HTML once it is not "show"ing?